engine/src/main/java/org/terasology/input/InputSystem.java

```diff
@@ -78,9 +77,6 @@
     @In
     private CameraTargetSystem targetSystem;
 
-    @In
-    private BindsSubsystem bindsSubsystem;
```

Comment:
The `bindsSubsystem` was `null` here anyways, and the only place this was used could easily be replaced with access to the super interface of `BindsManager`.